### PR TITLE
p2p rebooting the network

### PR DESCRIPTION
**Pull request by repo-alt**
Not sure why this flag was originally inverted, perhaps to prevent cycles in the old pathfinding algorithm
closes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/12905

### CODE REVIEW

**Comment by firenoo** [comment]
Before we merge it in let's make sure to test it in a nightly

**Comment by boubou19** [comment]
you'll have to replace the AE2 jar with the custom built version with the patch in the nightly, because nightlies autoupdate to latest tagged versions

**Comment by Hikari1nVoid** [comment]
idk if this one gets into the new nightly,if it is then it managed to crashed my ME network;

**Comment by chochem** [comment]
yes it is in 219-pre in the nightly https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/compare/rv3-beta-218-GTNH-pre...rv3-beta-219-GTNH-pre

**Review by firenoo — DISMISSED** [review]
Tested in dev env on the test world I sent you @repo-alt. On a some occasions, it triggers a ME system restart loop when the world loads. Since the one I've sent, I added some p2p connections with storage busses on them and a few interfaces.

**Comment by repo-alt** [comment]
> 

Can you resend? I tested on a large world I received from a player and it seemed to work fine.

**Comment by firenoo** [comment]
Looks like most of the bugs are fixed, however updating the network still causes a disruption. In MP this means that if 2 players are working on the same network, if one player is building out the network, the other cannot use the terminal.

Likely needs a change in Me terminal code.

**Comment by Dream-Master** [comment]
@repo-alt

File: src/main/java/appeng/me/cache/PathGridCache.java [hunk 1]
```diff
@@ -17,10 +17,7 @@
 import org.apache.logging.log4j.Level;
 
 import appeng.api.networking.*;
-import appeng.api.networking.events.MENetworkBootingStatusChange;
-import appeng.api.networking.events.MENetworkChannelChanged;
-import appeng.api.networking.events.MENetworkControllerChange;
-import appeng.api.networking.events.MENetworkEventSubscribe;
+import appeng.api.networking.events.*;
```
Comment (firenoo) [inline src/main/java/appeng/me/cache/PathGridCache.java:20]: nit: use explicit imports

Comment (repo-alt) [reply src/main/java/appeng/me/cache/PathGridCache.java:20]: well, blame spotless for that...

Comment (firenoo) [reply src/main/java/appeng/me/cache/PathGridCache.java:20]: changed the editorconfig

**Review by firenoo — APPROVED** [review]

**Pull request merged**